src/analytics.ts

```diff
@@ -342,43 +342,50 @@ export const logTiming = ({
 export const setScreenName = (
     screenName: string,
     params: {[key: string]: any} = {},
+    options?: LogEventOptions,
 ): Promise<void> => {
     if (!screenName) {
         console.warn('Missing analytics screenName');
         return Promise.resolve();
     }
 
+    const {sanitize = true} = {...defaultEventOptions, ...options};
     const previousScreenName = currentScreenName;
     currentScreenName = screenName;
 
+    const sanitizedParams = sanitize ? sanitizeAnalyticsParams(params) : params;
+    const sanitizedScreenName = sanitize
+        ? sanitizeAnalyticsParam(screenName)
+        : screenName;
```

Comment:
`screenName` shouln't be sanitized in any case. The `sanitize` option only applies to `params`